### ISSUE TITLE
[Bug]everyone can hear stuff like coffins or vampires drinking in my minecraft server

### ISSUE THREAD

**Issue by tdboy** [TeamLapen/Vampirism#643]
## Versions
- Minecraft:1.14.4                    
- Forge:28.2.0                       
- Vampirism:  1.6.0-beta.4                                           

## Issue Description
basically when ever a player vampires suck blood or go into coffin or I guess most actions with custom sounds everyone hears it loudly and upclose sounding even tho no one is near them can we please get a fix on that?

## Reproduce Steps
basically vampire on mc server suck blood or go in coffin everyone hears it loud as hell

## Additional Information
have none sorry lol

**Comment by maxanier** [comment]
Thank you for reporting. Apparently the new sound system cannot handle stereo sound files.
Will be fixed in next beta
